test/models.spec.ts

```diff
@@ -187,6 +189,74 @@ describe('Unit | Models', function () {
       // Assert
       testForExpectedMessage(errors, pageNameInvalidTypeMessage);
     });
+
+    it(`should return errors with one containing message '${permissionsInvalidMessage}' if permissions is not a number`, function () {
+      // Arrange
+      const testData = {
+        load: {
+          id: 'fakeId',
+          accessToken: 'fakeAccessToken',
+          permissions: "SomeString"
+        }
+      };
+
+      // Act
+      const errors = models.validateReportLoad(testData.load);
+
+      // Assert
+      testForExpectedMessage(errors, permissionsInvalidMessage);
+    });
+
+    it(`should return errors with one containing message '${permissionsInvalidMessage}' if permissions is invalid`, function () {
+      // Arrange
+      const testData = {
+        load: {
+          id: 'fakeId',
+          accessToken: 'fakeAccessToken',
+          permissions: 5
+        }
+      };
+
+      // Act
+      const errors = models.validateReportLoad(testData.load);
+
+      // Assert
+      testForExpectedMessage(errors, permissionsInvalidMessage);
+    });
+
+    it(`should return errors with one containing message '${viewModeInvalidMessage}' if permissions is not a number`, function () {
+      // Arrange
+      const testData = {
+        load: {
+          id: 'fakeId',
+          accessToken: 'fakeAccessToken',
+          viewMode: "ViewModeString"
+        }
+      };
+
+      // Act
+      const errors = models.validateReportLoad(testData.load);
+
+      // Assert
+      testForExpectedMessage(errors, viewModeInvalidMessage);
+    });
+
+    it(`should return errors with one containing message '${viewModeInvalidMessage}' if viewMode is invalid`, function () {
+      // Arrange
+      const testData = {
+        load: {
+          id: 'fakeId',
+          accessToken: 'fakeAccessToken',
+          viewMode: 5
+        }
+      };
+
+      // Act
+      const errors = models.validateReportLoad(testData.load);
+
+      // Assert
+      testForExpectedMessage(errors, viewModeInvalidMessage);
+    });
   });
 
```

Comment:
you may want to add two additional tests for success cases (to see that they don't return invalid message)